src/credentials/credentialsUtilities.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Credentials } from 'aws-sdk'
+
+export function asEnvironmentVariables(credentials: Credentials): NodeJS.ProcessEnv {
```

Comment:
this could go in `credentialsCreator.ts` instead of a new file.

Reply:
I'm not a fan of proliferating single-purpose files, but I also would like to avoid vaguely named "utility collection" files. I'll keep an eye out for consolidating functions in upcoming credentials provider related changes.